src/lit-html.ts

```diff
@@ -56,7 +56,11 @@ declare global {
 // IMPORTANT: do not change the property name or the assignment expression.
 // This line will be used in regexes to search for lit-html usage.
 // TODO(justinfagnani): inject version number at build time
-(window['litHtmlVersions'] || (window['litHtmlVersions'] = [])).push('1.1.2');
+const isBrowser = typeof window !== 'undefined'
+if (isBrowser) {
+  // If we run in the browser set version
+  (window['litHtmlVersions'] || (window['litHtmlVersions'] = [])).push('1.1.2');
```

Comment:
Rather than the `isBrowser` variable, let's just inline the check:

```suggestion
window ?? (window['litHtmlVersions'] || (window['litHtmlVersions'] = [])).push('1.1.2');
```

Reply:
typeof window !== 'undefined' is importent window ?? will not work as we then try to access window needs verification!

Reply:
(node:291371) ExperimentalWarning: The ESM module loader is experimental.
file:///home/frank/docker/apps/stealify/es/lit-html/lit-html-browser.mjs:49
window ?? (window['litHtmlVersions'] || (window['litHtmlVersions'] = [])).push('1.1.2');
        ^

SyntaxError: Unexpected token '?'
    at Loader.moduleStrategy (internal/modules/esm/translators.js:84:18)
    at async link (internal/modules/esm/module_job.js:36:21)

Reply:
Right, we do need the `typeof`. `??` should be compiled out by TypeScript with the `es2017` target, so I'm not sure why you would be getting a Node error.

Reply:
I'd still rather not use a ternary when the false branch returns `false`. It's redundant.

Reply:
I see, you did that. Thanks.